src/geo/leaflet/leaflet-map-view.js

```diff
@@ -45,8 +45,15 @@ var LeafletMapView = MapView.extend({
       // remove the "powered by leaflet"
       this.map_leaflet.attributionControl.setPrefix('');
 
+      // Disable scrollwheel
       if (this.map.get("scrollwheel") == false) this.map_leaflet.scrollWheelZoom.disable();
+      // Disable keyboard
       if (this.map.get("keyboard") == false) this.map_leaflet.keyboard.disable();
+      // Disable dragging (also doubleClickZoom)
```

Comment:
IMHO comments only makes sense if they explain the intention of a particular snippet that can't be captured in code itself (e.g answers _why does the code do this or that?_)